<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>952</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>CAN QT</string>
  </property>
  <widget class="QGroupBox" name="Test_File_groupBox_4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>291</width>
     <height>191</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>WRITE</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QLabel" name="label_65">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid transparent;
color: red;</string>
    </property>
    <property name="text">
     <string>Frame</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_CAN_Frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
	qproperty-cursorPosition: 0;
	border: 1px solid transparent;
}
QLineEdit::hover
{
	border: 4px solid green;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>Frame</string>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_66">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid transparent;
color: red;</string>
    </property>
    <property name="text">
     <string>Signal</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_CAN_Signal">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
	qproperty-cursorPosition: 0;
	border: 1px solid transparent;
}
QLineEdit::hover
{
	border: 4px solid green;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>Signal</string>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_67">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid transparent;
color: red;</string>
    </property>
    <property name="text">
     <string>Value</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_CAN_Value">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
	qproperty-cursorPosition: 0;
	border: 1px solid transparent;
}
QLineEdit::hover
{
	border: 4px solid green;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>Value</string>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_68">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid transparent;
color: red;</string>
    </property>
    <property name="text">
     <string>Timeout / Period</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_CAN_Period">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
	qproperty-cursorPosition: 0;
	border: 1px solid transparent;
}
QLineEdit::hover
{
	border: 4px solid green;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>Sec</string>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_CAN_Write">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: black;
	border-radius: 5px;
	border: 1px solid;
}
QPushButton::pressed{
background-color : blue; color: blue
}
QPushButton::hover{
background-color: lightblue;
}</string>
    </property>
    <property name="text">
     <string>WRITE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_CAN_Write_Period">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: black;
	border-radius: 5px;
	border: 1px solid;
}
QPushButton::pressed{
background-color : blue; color: blue
}
QPushButton::hover{
background-color: lightblue;
}</string>
    </property>
    <property name="text">
     <string>WRITE_P</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="Test_File_groupBox_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>461</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>CAN PERIOD SIGNAL</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QPushButton" name="btn_Read_CAN_Period_Sig">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: black;
	border-radius: 5px;
	border: 1px solid;
}
QPushButton::pressed{
background-color : blue; color: blue
}
QPushButton::hover{
background-color: lightblue;
}</string>
    </property>
    <property name="text">
     <string>READ PERIOD SIG</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Stop_CAN_Period">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: black;
	border-radius: 5px;
	border: 1px solid;
}
QPushButton::pressed{
background-color : blue; color: blue
}
QPushButton::hover{
background-color: lightblue;
}</string>
    </property>
    <property name="text">
     <string>STOP ALL PERIOD</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>571</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>LG Smart UI SemiBold</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CAN</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="Status_groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>151</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>STATUS</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QPushButton" name="btn_Connect">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: black;
	background-color: rgba(245, 188, 66, 150);
	border-radius: 5px;
	border: 1px solid transparent;
}
QPushButton::pressed{
background-color : blue; color: blue
}
QPushButton::hover{
background-color: lightblue;
}</string>
    </property>
    <property name="text">
     <string>CONNECT</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../img/usb.png</normaloff>../img/usb.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>21</width>
      <height>21</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_connect_status">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">qproperty-cursorPosition: 0;
border: 1px solid transparent;
background-color: rgba(255, 0, 0, 0.70)</string>
    </property>
    <property name="text">
     <string>Not Connected</string>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPlainTextEdit" name="pText_cmd">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>461</width>
     <height>91</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="plainText">
    <string>W: Write Msg (Frame, Signal, Value, Sec)
WP: Write Period Msg (Frame, Signal, Value, Sec)
R: Read Msg (Frame, Signal)
T: Time Delay (ex. T: 0.2)
Option: Display Option</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_send">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>410</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: black;
	background-color: rgba(66, 245, 102, 150);
	border-radius: 5px;
	border: 1px solid transparent;
}
QPushButton::pressed{
background-color : blue; color: blue
}
QPushButton::hover{
background-color: lightgreen;
}</string>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../img/send.png</normaloff>../img/send.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>13</width>
     <height>13</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_CAN_XCP">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: black;
	background-color:#E77FEC;
	border-radius: 5px;
	border: 1px solid transparent;
}
QPushButton::pressed{
background-color : blue; color: blue
}
QPushButton::hover{
background-color: lightblue;
}</string>
   </property>
   <property name="text">
    <string>XCP</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_69">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid transparent;
color: black;</string>
   </property>
   <property name="text">
    <string>Command</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_dbc_path">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">qproperty-cursorPosition: 0;
border: 1px solid transparent;</string>
   </property>
   <property name="text">
    <string>DBC Path</string>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_dbc_load">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>260</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: black;
	border-radius: 5px;
	border: 1px solid;
}
QPushButton::pressed{
background-color : blue; color: blue
}
QPushButton::hover{
background-color: lightblue;
}</string>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../img/load.png</normaloff>../img/load.png</iconset>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="pText_monitor">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>620</y>
     <width>461</width>
     <height>311</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_Rx_Rate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>580</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">qproperty-cursorPosition: 0;
border: 1px solid transparent;</string>
   </property>
   <property name="text">
    <string>Read Rate</string>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Rx_Read">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>580</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #42f566;
border: 1px solid black;</string>
   </property>
   <property name="text">
    <string>READ REAL-TIME</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_70">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>530</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid transparent;
color: black;</string>
   </property>
   <property name="text">
    <string>Monitor</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_Frame">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>580</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">qproperty-cursorPosition: 0;
border: 1px solid transparent;</string>
   </property>
   <property name="text">
    <string>Frame</string>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
